src/bodies.cpp

```diff
@@ -660,13 +660,35 @@ bool bodies::ConvexMesh::containsPoint(const Eigen::Vector3d &p, bool verbose) c
   if (bounding_box_.containsPoint(p))
   {
     Eigen::Vector3d ip(i_pose_ * p);
-    ip = mesh_data_->mesh_center_ + (ip - mesh_data_->mesh_center_) * scale_;
+    ip = mesh_data_->mesh_center_ + (ip - mesh_data_->mesh_center_) / scale_;
```

Comment:
Just to be clear: This will scale the mesh UP, i.e. a scale of 1.5 will make it appear 1.5 times larger.  This is what we want, right? If I understand what's happening here correctly, it transforms the coordinates to the internally unscaled mesh, thus they need to be scaled down.

I believe the main reason this was never discovered is that the bounding_box check happens before. The bounding_box is already scaled up and thus catches many cases.